manifests/postgres/sql/create-run-databases.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
+CREATE DATABASE run_database;
+\c run_database;
+
+CREATE SCHEMA run_schema;
+
+CREATE TABLE run_schema.run_registry_meta(
+    run_number INT PRIMARY KEY NOT NULL,
+    start_time TIMESTAMP (6) NOT NULL,
+    stop_time TIMESTAMP (6),
+    detector_id VARCHAR (40) NOT NULL,
+    run_type VARCHAR (40) NOT NULL,
+    filename VARCHAR (100) NOT NULL,
+    software_version VARCHAR (40)
+);
+
+CREATE UNIQUE INDEX run_registry_meta_pk ON run_schema.run_registry_meta (run_number);
+
+CREATE TABLE run_schema.run_registry_configs(
+  run_number INT NOT NULL,
+  configuration BYTEA NOT NULL,
+  CONSTRAINT run_registry_meta_pk
+    FOREIGN KEY(run_number)
+       REFERENCES run_schema.run_registry_meta(run_number)
+);
+
+CREATE TABLE run_schema.run_number(
+  rn INT PRIMARY KEY NOT NULL,
+  start_time TIMESTAMP (6) NOT NULL,
+  flag BOOLEAN NOT NULL,
+  stop_time TIMESTAMP (6)
+);
+
+INSERT INTO run_schema.run_number(rn, start_time, flag, stop_time) VALUES (1000000, CURRENT_TIMESTAMP, true, CURRENT_TIMESTAMP);
```

Comment:
It occurs to me that it'd be nice to have the first run number to be configurable. Maybe this isn't for this PR, but a nice-to-have?

Reply:
We are no longer using this bash script right now unless I figure out how to run the the sql file as a psql command in the bash script.

Reply:
Right now we are not inserting any data into the runnumber db via the configmap

Reply:
If we are not using this file, please delete it.

Reply:
Addressed here 0a6411ab5a241072df8633a660540f5dcf424bee